backend/migrations/20210210112014_refactor_for_lang.sql

```diff
@@ -0,0 +1,34 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE books DROP COLUMN read_video;
+ALTER TABLE books DROP COLUMN read_body;
+ALTER TABLE books DROP COLUMN explore_video;
+ALTER TABLE books DROP COLUMN explore_body;
+ALTER TABLE books DROP COLUMN learn_video;
+ALTER TABLE books DROP COLUMN learn_body;
```

Comment:
read_body, explore_body, and learn_body need a `NOT NULL` constraint or you will get an error trying to scan a null into a `TabContent.Body` which is a string